python/cudf/bindings/types.pxd

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cudf.bindings.cudf_cpp cimport *
+
+
+cdef extern from "table.hpp" namespace "cudf" nogil:
+    
+    cdef cppclass table:
+
+        table(gdf_column* cols[], gdf_size_type num_cols) except +
+
+
+        gdf_column** begin() except +
+
+        gdf_column** end() except +
+
+        gdf_column* get_column(gdf_index_type index) except +
+ 
+        gdf_size_type num_columns() except +
+
+        gdf_size_type num_rows() except +
+
+# Todo? add const overloads 
```

Comment:
Why not add them?

Reply:
Since it causes cython error:

> Error compiling Cython file:
>         const gdf_column* get_column(gdf_index_type index) **const** except +
>  
> cudf/bindings/types.pxd:33:65: Syntax error in C variable declaration
> Error compiling Cython file:

Reply:
Probably due to const.

Reply:
Cython doesn't support east const which is likely the reasoning for it failing to compile